lib/nine_one_one/notifier.rb

```diff
@@ -0,0 +1,36 @@
+module NineOneOne
+  class Notifier
+    def initialize(config)
+      @config = config
+    end
+
+    def emergency(incident_key, description, details_hash = nil)
+      emergency_service.trigger_event(incident_key, description, details_hash)
+    end
+
+    def notify(message)
+      notification_service.notify(message)
+    end
+
+    def emergency_service
+      if config.send_pagers
```

Comment:
for me it's not like either pager or log, it's rather "a pager or not" + log
maybe log could be configurable as well

I know it's not part of this PR so feel free to ignore it.

Reply:
I agree with both statements ;)